File: .repos/nexmo-community/nexmo-java-quickstart/src/main/java/com/nexmo/quickstart/initialize/FullAuth.java
package com.nexmo.quickstart.initialize;

import com.nexmo.client.NexmoClient;
import com.nexmo.client.auth.AuthMethod;
import com.nexmo.client.auth.JWTAuthMethod;
import com.nexmo.client.auth.TokenAuthMethod;

import java.nio.file.FileSystems;

import static com.nexmo.quickstart.Util.envVar;


/**
 * Example of configuring a NexmoClient with authentication for
 * both API secret and Application (JWT) authentication credentials.
 */
public class FullAuth {
    public static void main(String[] argv) throws Exception {
        String NEXMO_API_KEY = envVar("API_KEY");
        String NEXMO_API_SECRET = envVar("API_SECRET");
        String NEXMO_APPLICATION_ID = envVar("APPLICATION_ID");
        String NEXMO_APPLICATION_PRIVATE_KEY_PATH = envVar("PRIVATE_KEY");

        AuthMethod tokenAuth = new TokenAuthMethod(NEXMO_API_KEY, NEXMO_API_SECRET);
        AuthMethod applicationAuth = new JWTAuthMethod(
                NEXMO_APPLICATION_ID,
                FileSystems.getDefault().getPath(NEXMO_APPLICATION_PRIVATE_KEY_PATH)
        );
        NexmoClient client = new NexmoClient(tokenAuth, applicationAuth);
    }
}

File: .repos/nexmo-community/nexmo-dotnet-quickstart/NexmoDotNetQuickStarts/Authentication/FullAuth.cs
using Nexmo.Api;

namespace NexmoDotNetQuickStarts.Authentication
{
    /// <summary>
    /// This class is only used for the purpose of building documentation on https://developer.nexmo.com/
    /// It is not used in any of the code samples provided in these Quickstarts
    /// </summary>
    public class FullAuth
    {
        public FullAuth()
        {
            var client = new Client(creds: new Nexmo.Api.Request.Credentials
            {
                ApiKey = "NEXMO_API_KEY",
                ApiSecret = "NEXMO_API_SECRET",
                ApplicationId = "NEXMO_APPLICATION_ID",
                ApplicationKey = "NEXMO_APPLICATION_PRIVATE_KEY"
            });
        }
    }
}